src/quoridor/QuoridorGame.cs

```
@@ -0,0 +1,56 @@
+﻿using QuoridorEngine.src.solver;
+
+namespace QuoridorEngine.src.quoridor
+{
+    internal class QuoridorGame : IGameState
+    {
+        /// <summary>
+        /// Returns true if the state is terminal (i.e. game is over)
+        /// </summary>
+        /// <returns>True if the state is terminal (i.e. game is over)</returns>
+        public bool IsTerminalState()
+        {
+            return false;
```

Maybe throw a NotImplemented exception for all the functions that are not yet doing anything so we don't accidentally forget to fill out any. Applies to other files and functions in this folder as well,